openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -58,11 +58,31 @@ def __init__(self, course_key, request, username=''):
             course_key,
             staff_access=self.is_staff,
         )
-        self.course_masquerade = course_masquerade
+        self._course_masquerade = course_masquerade
 
     def __getattr__(self, name):
         return getattr(self.overview, name)
 
+    @property
+    def course_masquerade(self):
+        """
+        Return details about current masquerade status
```

Comment:
TODO: I have some un-pushed code that we _may_ keep (to include a display name).

In theory, we could probably parse it out on the frontend, but the MFE would just be parsing data the API already spit out anyway, so I _think_ it makes sense to consolidate here.

I'll circle back pre-hand-off.